technical update to Feature on Demand (#4773)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/administration/server-core/what-is-server-core.md

- No OOBE (out-of-box-experience) for setting up Server Core
- No audio support

The following table shows which applications are available *locally* on Server Core vs Server with Desktop Experience. **Important**: In most cases, applications that are listed as "not available" below can be run remotely from a Windows client computer or [Windows Admin Center](../../manage/windows-admin-center/overview.md) to manage your Server Core installation. 

### Versioning and Downloads of Feature on Demand for Server Core

The list below reflects the FoD level as of Version 1809 (Server 2019 LTSC). 
You can greatly enchance your Server Core compatibility by applying a FoD package, newer than 1809.

> *Download locations*

Reviewer: This line looks like it should have been a subheading for the previous H3 heading, so perhaps use a H4 heading here?
```suggestion
#### Download locations
```

> Later Windows Server (SAC) FoD, can be obtained in your specific [MS Licensing Center](https://businessaccount.microsoft.com/), if you have active Software Assurance.

Reviewer: ```suggestion
Later Windows Server (SAC) FoD can be obtained from your specific [MS Licensing Center](https://businessaccount.microsoft.com/), if you have got active Software Assurance.
```

> You'll need a Microsoft Work or School Account enrolled to VLSC or Business Center, to access the FoD download.

Reviewer: ```suggestion
You'll need a Microsoft Work or School Account enrolled to VLSC or Business Center to access the FoD download.
```

Alternatively, you might have access to a [Visual Studio Subscription](https://my.visualstudio.com/).
You'll need a Microsoft Home, Work or School account, enrolled to an active Visual Studio subscription, to access the FoD download.

> Latest Server Insider FoD. This is downloadable at [Server Insider webpage](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewserver). 

Reviewer: ```suggestion
Latest Server Insider FoD. This is downloadable at the [Server Insider webpage](https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewserver). 
```
